opennms-config/src/main/java/org/opennms/netmgt/config/TrapdConfigFactory.java

```diff
@@ -196,14 +196,14 @@ public synchronized boolean getNewSuspectOnTrap() {
     @Override
     public synchronized List<SnmpV3User> getSnmpV3Users() {
         List<SnmpV3User> snmpUsers = new ArrayList<SnmpV3User>();
-        for (Snmpv3User user : m_config.getSnmpv3UserCollection()) {
+        for (SnmpV3User user : m_config.getSnmpv3UserCollection()) {
             snmpUsers.add(new SnmpV3User(
                     user.getEngineId(),
                     user.getSecurityName(),
                     user.getAuthProtocol(),
-                    user.getAuthPassphrase(),
-                    user.getPrivacyProtocol(),
-                    user.getPrivacyPassphrase()));
+                    user.getAuthPassPhrase(),
+                    user.getAuthProtocol(),
```

Comment:
Something is wrong here, you should still call the getPrivacyProtocol() and getPrivacyPassphrase() methods here.